Task16/RealPower.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task16
+{
+    public struct RealPower
+    {
+        public double basse;
+        public double exponent;
+
+        public double Basse
```

Comment:
В задании другое имя
```suggestion
        public double Base
```